src/icons/raw/Check.svg

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I recommend starting with a React element, and not an svg file (you can see the tooltip pr for reference).
That way we won't need to deal with the whole svg build phase.

Reply:
are you sure? @shlomitc told be to use `svg2react-icon`

Reply:
@eranshabi 
Yeah, lets start with working on the Button itself, and will add the icon support later

Reply:
In general - this icons looks like a mess. A lot of unnecessary meta data and wrong icon structure. You should follow these guidelines 
https://github.com/wix/wix-ui/blob/master/packages/wix-ui-icons-common/README.md#guidelines

Reply:
Hey I don't think this needs a fix because it's only the raw svg the ux lady gave me. You can see in the `dist` folder after you build the project that all the unnecessary meta data will not get into production.